doc/help.eus/makehelp.l

```diff
@@ -60,7 +60,7 @@
 			      "jmanipulator"))
 
 (defun addition-to-help(fp2 tex-dir fname)
-  (let ((fp (open (make-pathname :directory (parse-directory (pathname tex-dir))
+  (let ((fp (open (make-pathname :directory (list tex-dir)
```

Comment:
@Affonso-Gui Do we have Makefile target to run makehelp.l script?

Reply:
Not yet. Maybe I can try to add it
https://github.com/euslisp/EusLisp/issues/322